main.sh

```diff
@@ -355,6 +356,11 @@ register_binfmt() {
 
 setup_linux_host() {
     apt_packages=()
+    while read -rd,; do
+        if [[ -n "${REPLY}" ]]; then
+            apt_packages+=("${REPLY}")
+        fi
+    done <<<"${packages// /,},"
```

Comment:
Also, the following change should allow tab/line separated cases.
```suggestion
    done <<<"${packages//[$' \t\n']/,},"
```